src/meta-srv/src/service/admin.rs

```diff
@@ -163,7 +163,8 @@ where
             })
             .unwrap_or_default();
         let path = req.uri().path().to_owned();
-        Box::pin(async move { router.call(&path, query_params).await })
+        let method = req.method().to_owned();
```

Comment:
Prefer to use `clone()`.
```suggestion
        let method = req.method().clone();
```

Reply:
Updated.